showcase/src/assets/translations/en.json

```diff
@@ -87,6 +87,17 @@
       "LOGOUT": {
         "LOGOUT_DEFAULT_ICON": "Logout button - Default icon",
         "LOGOUT_CUSTOM_ICON": "Logout button - Custom icon"
+      },
+      "COLLAPSIBLE": {
```

Comment:
I think it is good practice to order this json alphabetically :-) (even if we didn t do it 100% for now)